lib/freedom_patches/better_handlebars_errors.rb

```diff
@@ -1,13 +1,13 @@
 module Ember
   module Handlebars
-    class Template < Tilt::Template
```

Comment:
Tile dependency has been dropped in https://github.com/tricknotes/ember-handlebars-template/commit/66ec5ef190f72b4154905cf346e01ae389a95444